src/main/java/ch/njol/skript/registrations/Classes.java

```diff
@@ -292,7 +292,7 @@ private static void checkAllowClassInfoInteraction() {
 	}
 	
 	/**
-	 * Gets the class info of the given class or its closest registered superclass. This method will never return null unless <tt>c</tt> is null.
+	 * Gets the class info of the given class or it's closest registered superclass. This method will never return null unless <tt>c</tt> is null.
```

Comment:
https://www.dictionary.com/e/its-vs-its/

Reply:
It's is used in other parts of the file. So.

Reply:
It's is a contraction of `it is/has`. It, however, is the possessive form of the pronoun it — and is used as such here.

If `it's` is incorrectly used somewhere else in the code, then we should fix those instead, because this one was already correct